extension/chrome/elements/composer/composer-input.ts

```diff
@@ -137,13 +138,30 @@ export class ComposerInput extends ComposerComponent {
   }
 
   private resizeReplyBox = () => {
-    this.squire.addEventListener('input', () => {
+    this.squire.addEventListener('cursor', () => {
       if (this.composer.view.isReplyBox) {
         this.composer.size.resizeComposeBox();
```

Comment:
Possible concern - in case `cursor` event spams with each keypress, this would probably cause higher CPU usage to evaluate if it should resize or not on each keypress (I've had that problem before). I'll have a look to confirm.

Reply:
It does spam but could not observe any difference in cpu usage (70% while typing, either way).